pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -68,7 +62,7 @@ type provider interface {
 type Service struct {
 	ctx             context
 	store           storage.Store
-	callbackChannel chan didCommChMessage
+	callbackChannel chan string
```

Comment:
it's now hard to understand what is being sent on this `callbackChannel`, since all the code has is `chan string`.

Reply:
From discussion, it sounds like this needs to be a struct anyways to internally signal Continue vs Fail.

Reply:
[Reverted] the changes. Also added a [TODO](https://github.com/hyperledger/aries-framework-go/pull/437/files#diff-f761c91e753eba1c42a68076df06ce12R333)